src/components/attendee/AttendeeLayout.tsx

```diff
@@ -32,9 +32,9 @@ const _AttendeeLayout: React.FC<_AttendeeLayoutProps> = ({ userId }) => {
   const [backButtonSpace, setBackButtonSpace] = useState<SpaceWithId>();
 
   useEffect(() => {
-    document.documentElement.classList.add(styles.html);
+    document.documentElement.classList.add("AttendeeLayout", styles.html);
```

Comment:
I don't think there is a need of adding more classes to the body element (a god citizen of app would just not get wider from the screen). 

The idea here is if there is a component that goes wider than the parent component, the parent component should display scroll bars, not rely on the body element to do it.

Reply:
I don't quite get your comment here. As far as I understand the only way to remove scroll from the body element, when another modal is up, is to remove the `overflow` off it, so here's why I'm adding this classname to use in the hook I've added in this PR